src/block_cache/block_chunkcache.c

```diff
@@ -738,6 +738,78 @@ __wt_chunkcache_remove(
     return (0);
 }
 
+/*
+ * __wt_chunkcache_ingest --
+ *     Read all the contents from a file and insert it into the chunkcache.
+ */
+int
+__wt_chunkcache_ingest(
+  WT_SESSION_IMPL *session, const char *local_name, const char *sp_obj_name, uint32_t object_id)
+{
+    WT_CHUNKCACHE *chunkcache;
+    WT_CHUNKCACHE_CHUNK *chunk;
+    WT_CHUNKCACHE_HASHID hash_id;
+    WT_DECL_RET;
+    WT_FH *fh;
+    size_t already_read, size;
+    uint64_t bucket_id;
+
+    chunkcache = &S2C(session)->chunkcache;
+    already_read = 0;
+
+    WT_RET(__wt_open(session, local_name, WT_FS_OPEN_FILE_TYPE_DATA, WT_FS_OPEN_READONLY, &fh));
```

Comment:
We should check that both the chunk cache is configured, and we've enabled flushed data insertion.

Reply:
This gets checked before we call this function.

Reply:
Right - my point is that the better place to check it is here.

Reply:
done here  [fedd294](https://github.com/wiredtiger/wiredtiger/pull/9677/commits/fedd2945bfb673c4d64aaeaece7f4e060fe6d71d)